### DISCUSSION TITLE
Stamped_Core

### DISCUSSION THREAD

**Discussion by eneskirca** [breezefront/community#104] [3. Integrations]
I am writing to inquire about the compatibility of the Stamped_Core Magento extension with the Breeze Theme.

As an e-commerce owner, we are using for collecting reviews and showing reviews on the product page is the Stamped_Core extension which integrates with stamped.io, which allows us to collect customer reviews and ratings. We have found this app to be a valuable asset for our business, and it is important for us to ensure that it is compatible with the Breeze Theme.

We would greatly appreciate any information or recommendations you can provide regarding the compatibility of the Stamped_Core Magento extension with the Breeze Theme. We believe that the Breeze Theme is a great fit for our website and are eager to switch to this theme if it is compatible with the Stamped_Core extension.

**Comment by eneskirca** [comment]
Would like help with third-party modules integrations with Breeze module.

.  Stamped_Core  ( it is stamped.io's review app magento extension )

Thank you for any help!

**Comment by vovayatsyuk** [comment]
Module source code: https://github.com/Stamped-io/Magento-v2.x

They use inline `require` call: https://github.com/Stamped-io/Magento-v2.x/blob/master/view/frontend/templates/stamped/javascript.phtml#L5

The rest of the code seems compatible with Breeze, so I think the only change we need to apply is to add `data-breeze` attribute to the `<script>` tag:

```js
<script data-breeze>
    require([....
```

Can you try if it will work?
